server/config/config.go

```diff
@@ -21,6 +22,9 @@ const (
 	DefaultGRPCWebAddress = "0.0.0.0:9091"
 )
 
+// ErrSetMinGasPrice defines an error occurred if min-gas-prices field in BaseConfig is empty.
+var ErrSetMinGasPrice = errors.New("please set min gas price in app.toml or flag or env var")
```

Comment:
I think we should have a general error in `errors`:
```
ErrConfig = Register(RootCodespace, xxx, "error in config.toml")
``` 

and then use:

```
errors.ErrConfig.Wrap("set min gas price in app.toml or flag or env variable")
```

Reply:
Done.